WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,5 +1,11 @@
-* [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
-* [*] Stats: Scroll to new Insights card when added. [#17894]
-* [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [***] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
-* [*] Block editor: Additional error messages for media upload failures. [#17971]
+Three, two, one, liftoff—the Jetpack app now allows you to use and create WordPress.com sites and use the same Reader features as the WordPress app. The sky’s the limit.
```

Comment:
I got confused by the "use" verb.

#17914 says: "👁️ Notice on the site switcher all of your connected sites are displayed, not just Jetpack ones"

Maybe a clearer wording would be:

> use all of your WordPress.com sites and create new ones, too

But, then we'd have _another_ and conjunction "use all ... and create ... and use the same ..." I think that would make the sentence worse off.

So, all in all, I think this is fine to ship with.

Reply:
What about this?
```suggestion
Three, two, one, liftoff—the Jetpack app now allows you to use all of your WordPress.com sites, create new ones, and use the same Reader features as the WordPress app. The sky’s the limit.
```